README.md

```diff
@@ -4,32 +4,26 @@
 [![Documentation Status](https://readthedocs.org/projects/viziphant/badge/?version=latest)](https://viziphant.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Since Travis-CI is no longer used, remove the Build Status badge from here?

Reply:
Yes, definitely.